modoptions.lua

```diff
@@ -505,6 +505,19 @@ local options = {
         section	= "options",
     },
 
+    {
+        key		= "wreckage_metal_ratio",
+        name	= "Unit Wreck Metal Percentage",
+        desc	= "Percent of unit metal that is left in its wrecks (and heaps)",
+        hidden 	= true,
+        type	= "number",
+        section	= "options",
+        def		= 0.6,
+        min		= 0, -- see usage in alldefs_post for actual bounds
```

Comment:
Actual bounds seem to be 0-1 too now so afaict the comment also became redundant.